kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -22,17 +22,25 @@ This script generates the RCU schema based `schemaPrefix` and `dburl`.
 The script assumes that either the image, `container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4`, is available in the Docker image repository or an `ImagePullSecret` is created for `container-registry.oracle.com`. To create a secret for accessing `container-registry.oracle.com`, see the script `create-image-pull-secret.sh`.
 
 ```
-$ ./create-rcu-schema.sh -h
-usage: ./create-rcu-schema.sh -s <schemaPrefix> -t <rcuType> -d <dburl> -i <image> -s <dockerstore> [-h]
+$ ./create-rcu-schema.sh  -h
+usage: ./create-rcu-schema.sh -s <schemaPrefix> -t <schemaType> -d <dburl> -i <image> -p <docker-store> -n <namespace> -q <sysPassword> -r <schemaPassword>  -o <rcuOutputDir>  [-h]
   -s RCU Schema Prefix (required)
   -t RCU Schema Type (optional)
-      Supported values: fmw(default),soa,osb,soaosb,soaess,soaessosb
+      (supported values: fmw(default), soa, osb, soaosb, soaess, soaessosb)
   -d RCU Oracle Database URL (optional)
       (default: oracle-db.default.svc.cluster.local:1521/devpdb.k8s)
```

Comment:
Are we okay that the database is still assumed to run in the default namespace? I'm guessing that it's a "yes" as we already provide an option to replace this URL.

Reply:
Yes, default namespace is also considered default value for namespace.